roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -9,6 +9,8 @@
   failed_when: false
   always_run: true
   register: osd_id
+  until: osd_id.stdout_lines|length == devices|length
```

Comment:
Hmm, I noticed you use `devices|unique` in a couple of places. Should I change this to `devices|unique|length`?

Reply:
Yes please.